src/service/stripe-service.ts

```diff
@@ -46,6 +48,17 @@ export default class StripeService {
     this.logger = getLogger('StripeController');
   }
 
+  /**
+   * Topup should be at least 10 euros or the users negative balance.
+   * @param balance
+   * @param request
+   */
+  public static validateStripeRequestAmount(balance: BalanceResponse, request: StripeRequest): boolean {
+    // Check if top up is enough
+    if (request.amount.amount > (process.env.MIN_TOPUP || 1000)) return true;
+    return request.amount.amount === balance.amount.amount;
```

Comment:
Isn't your balance negative and the to-be-deposited amount positive? Meaning that this will not work for negative balances? Still confused though, as there are test cases that seem to work?